static/js/tools/statsvar_menu.tsx

```diff
@@ -95,21 +97,22 @@ class Node extends Component<NodePropType, NodeStateType> {
     }
 
     return (
-      <ul className="noborder">
-        <li className="value" id={this.props.l}>
-          <span>
-            <a className="value-link">
-              {this.props.l + "  "}
-              <sup>
-                {this.props.c !== 0 && "(" + this.props.c + ")"}
-              </sup>
-              {checkboxImg}
-              {expandImg}
-            </a>
-          </span>
-          {child}
-        </li>
-      </ul>
+      (typeof checkboxImg !== "undefined" ||
```

Comment:
better to add a boolean class variable to record this than check the undefined. Can set the boolean around line 81

Reply:
done.